pom.xml

```diff
@@ -209,19 +230,49 @@
             </exclusions>
         </dependency>
     </dependencies>
-
     <build>
         <pluginManagement>
             <plugins>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-checkstyle-plugin</artifactId>
-                    <version>2.17</version>
+                    <version>${maven-checkstyle-plugin.version}</version>
```

Comment:
We shouldn't use checkstyle within the build chain because it points at a local xml which has been centralised meaning the localised versions will fall out of date. Suggest removing the checks from the pom entirely and rely on the job in the pipeline instead.

Reply:
Guide on how to get checkstyle working in your IDE [here](https://companieshouse.atlassian.net/wiki/spaces/JU/pages/4887281743/Companies+House+Style+guides)

Reply:
done